docs/prover/cli/options.md

```diff
@@ -949,7 +953,7 @@ This option disables the storage splitting optimization.
 
 
 (--allow_solidity_calls_in_quantifiers)=
-### --allow_solidity_calls_in_quantifiers
+### `--allow_solidity_calls_in_quantifiers`
```

Comment:
an aside - if you're faster than me... can you fix to `--prover_args` instead of `--settings` in line 938 `--settings -smt_groundQuantifiers=false`? just noticed it's out of date...

Reply:
hm, the accent seems to be on master already; changed `--settings` (in separate branch, PR upcoming ..)